src/index.tsx

```
@@ -1,13 +1,17 @@
+import React from 'react';
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import { HashRouter } from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './App.scss';
 
 import { App } from './App';
 
-createRoot(document.getElementById('root') as HTMLDivElement).render(
-  <Router>
+const root = createRoot(document.getElementById('root') as HTMLDivElement);
```

`document.getElementById('root')` is cast directly to `HTMLDivElement`. If the element is missing, this hides the null case and can make debugging harder. Consider a runtime null-check so you can throw a clear error message or use a non-null assertion if you're certain the element exists.